### PR TITLE
Adding links to Table, List and Card view about Loading State.

### LINKED ISSUE #467: Add guidance for a loading state to the Card View
This is an enhancement to an existing pattern to add a loading state note to the card view.

When there are performance issues, users have to wait for the view to load.  We should give guidance around using a loading state ( which would likely be a Large Spinner control with the "Loading" text below it).

Note that on the design page of this pattern, we are already giving guidance around if the view is empty http://www.patternfly.org/pattern-library/content-views/card-view/#/design.

- Stakeholders @serenamarie125 @jennyhaines

@mcarrano @maryclarke @catrobson do you support this suggestion?

Associated Issues: 
https://github.com/patternfly/patternfly-design/issues/468
https://github.com/patternfly/patternfly-design/issues/469

### PR DESCRIPTION
**Pull request by terezanovotna**
## Description
* added links to Table, List, and Card view about Loading State
*closes #467 #468 #469 

## Contribution Submission Checklist

- [x] A GitHub issue was created and reviewed with the PatternFly community.
- [x] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [x] Any new files have been added to the folder for that pattern.
- [x] The folder, for brand new patterns, has been titled with the name of the pattern.
- [x] All images are in an "img" folder and embedded images have the correct filepath.
- [x] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

**Comment by terezanovotna** [comment]
While adding the note under Additional Notes about Behavior, I also alphabetized the bullet points.

File: pattern-library/content-views/table-view/design/design.md [hunk 1]
```diff
@@ -40,4 +41,4 @@
 - Ability to expand and collapse rows to give user the option to view more details on each item.
 - Multi-column sort.
 - Sticky column headers.
-- Mobile design.
\ No newline at end of file
+- Mobile design.
```
Comment (mcarrano) [inline pattern-library/content-views/table-view/design/design.md]: Not sure what was changed here, but when I preview the file, this last section no longer renders as a bulleted list.

File: pattern-library/content-views/table-view/design/design.md [hunk 2]
```diff
@@ -32,6 +32,7 @@
 
 ## Additional Notes about Behavior
 - **Empty State:** If no items exist in the table, display the [Empty State](http://www.patternfly.org/pattern-library/communication/empty-state/) pattern. If there are no items to display as the result of a null filter result, see the [Filter](http://www.patternfly.org/pattern-library/forms-and-controls/filter/) pattern for more information about how to display this.
+- **Loading State:** If content is loading, display the [Loading State](http://www.patternfly.org/pattern-library/communication/loading-state/) pattern.
 - **Pagination:** Table View supports pagination. See [Pagination](https://www.patternfly.org/pattern-library/navigation/pagination/) for more details.
 
 <br>
```
Comment (mcarrano) [inline pattern-library/content-views/table-view/design/design.md]: The content below here is not rendering correctly in the markdown preview in GitHub. @LHinson @cliffpyles any idea why?

Comment (terezanovotna) [reply pattern-library/content-views/table-view/design/design.md]: @mcarrano added empty row and now it works

**Comment by mcarrano** [comment]
I will go ahead and merge this one. Thanks @terezanovotna !

**Pull request merged**